testbook/client.py

```diff
@@ -251,9 +251,14 @@ def value(self, code: str) -> Any:
 
         try:
             outputs = self.inject(inject_code, pop=True).outputs
+
+            if outputs[0].output_type == "error":
+                # will receive error when `allow_errors` is set to True
+                raise RuntimeError('serialize error')
```

Comment:
Probably shouldn't reuse RuntimeError for this since an actual runtime error in self.inject would be mistakenly reused. Not sure the case this is supposed to cover off-hand

Reply:
Oh yeah, it should be `TestbookRuntimeError`. Will make that change.